backend/src/controllers/EventController.ts

```diff
@@ -230,6 +231,25 @@ export class EventController {
     return this.attendanceMapper.entityToResponse(newAttendance);
   }
 
+  @Get('/:eventID/signin/affiliate/result')
+  @ResponseSchema(AttendanceResultResponse)
+  @OpenAPI({ security: [{ TokenAuth: [] }] })
+  async affiliateEventSigninResult(
```

Comment:
would be good to name functions as verbs, looks like this gets the attendance for an affiliate event?

```suggestion
  async getAffiliateEventAttendance
```